### PR TITLE
Add owners table

### PR DESCRIPTION
**Pull request by tingamapuro04**

### CODE REVIEW

**Comment by tingamapuro04** [comment]
I have worked on the query branch and I have run the queries required and they are included in the query.sql and data.sql. Below are the screenshots of the results of the various queries
winchester
first_lady
pokemon
pokemons
species_grouped
total

File: schema.sql [hunk 1]
```diff
@@ -11,3 +11,34 @@ CREATE TABLE animals (
 
 ALTER TABLE animals
 ADD species TEXT;
+
+CREATE TABLE owners (
+  id SERIAL PRIMARY KEY,
+  full_name TEXT,
+  age INT
+);
```
Comment (mateo951) [inline schema.sql:19]: Awesome! You added a new table with the name `owners` with the proper columns 🚀

File: schema.sql [hunk 2]
```diff
@@ -11,3 +11,34 @@ CREATE TABLE animals (
...
+
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(15)
+);
```
Comment (mateo951) [inline schema.sql:24]: Awesome! You added a new table with the name species with the proper columns 🚀

File: schema.sql [hunk 3]
```diff
@@ -11,3 +11,34 @@ CREATE TABLE animals (
...
+ALTER TABLE animals
+ADD FOREIGN KEY (species_id) REFERENCES species(id)
+ON DELETE SET NULL;
+
+ALTER TABLE animals
+ADD owner_id INT;
+
+ALTER TABLE animals
+ADD FOREIGN KEY (owner_id) REFERENCES owners(id)
+ON DELETE SET NULL;
```
Comment (mateo951) [inline schema.sql:44]: Amazing work! You modify the `animals` table as expected ✔

File: data.sql [hunk 4]
```diff
@@ -8,3 +8,33 @@ INSERT INTO animals VALUES
...
+
+
+INSERT INTO owners(full_name, age) VALUES
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO species(name) VALUES('Pokemon'), ('Digimon');
```
Comment (mateo951) [inline data.sql]: Nicely done! Please don't forget to add the first owner to your table 🤓

```sql
 INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);
```

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @tingamapuro04, 

🔥 Highlights 🔥
- Answered queries 🐐
- Table modifications 🐐

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by tingamapuro04** [comment]
I have added the first owner as suggested

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @tingamapuro04 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 
- [ optional] please always add a pull request description as it helps to know what you did in the project

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**